sacred/experiment.py

```diff
@@ -54,6 +55,7 @@ def __init__(
         ingredients: Sequence[Ingredient] = (),
         interactive: bool = False,
         base_dir: Optional[PathType] = None,
+        additional_cli_options: Sequence[CLIOption] = None,
```

Comment:
```suggestion
        additional_cli_options: Optional[Sequence[CLIOption]] = None,
```

Reply:
Thanks, I wish there was a way to avoid typing `Optional` all the time when the default value is `None`.